packages/core/crates/core-hopr/src/adaptors/network.rs

```diff
@@ -5,6 +5,7 @@ use core_network::{
 };
 use core_strategy::generic::PeerQuality;
 use futures::channel::mpsc::Sender;
+use utils_misc::time::native::current_timestamp;
```

Comment:
Please keep here the cfg fork for wasm and non-wasm `use`. Otherwise it will crash in the wasm context. 
```suggestion
#[cfg(any(not(feature = "wasm"), test))]
use utils_misc::time::native::current_timestamp;

#[cfg(all(feature = "wasm", not(test)))]
use utils_misc::time::wasm::current_timestamp;

```